examples/Bookshelf/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Tue Jun 01 17:22:02 BST 2021
+distributionBase=GRADLE_USER_HOME
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.0-bin.zip
```

Comment:
Would be nice if we could align the version of gradle throughout all the projects. Every version will trigger its own daemon. But I guess only option is sym-linking the file, so maybe better off to leave them. Just have in mind that we are running many different versions as part of CI build, etc.